### DISCUSSION TITLE
why do we need to record the active date when the user's status is banned or not yet activated?

### DISCUSSION THREAD

**Discussion by warcooft** [codeigniter4/shield#1179] [General]
I'm curious, why do we need to record the active date when the user's status is banned or not yet activated? Wouldn't it be better to move it to the bottom, so that the last active date is recorded only for users who are truly active?
https://github.com/codeigniter4/shield/blob/60f1ae3d7f8bf88702d015139acc618e8a24acc6/src/Filters/SessionAuth.php#L54-L80

**Comment by kenjis** [comment]
Yes, it seems reasonable to update active date only for users activated and not banned.
